docs/user/config/tsConfig.md

```diff
@@ -48,7 +48,7 @@ module.exports = {
 
 #### Inline compiler options
 
-Refer to the [TypeScript compiler options](https://www.typescriptlang.org/docs/handbook/compiler-options.html) to know what can be used.
+Refer to the TypeScript [compiler options][] for reference.
```

Comment:
why do you remove the reference link ?

Reply:
It’s not removed. It’s moved to the bottom of the page, because it’s referenced 3x in this document.